Fundamentals/Setup/Install/install-umbraco-with-vs-code-v9.md

```diff
@@ -1,45 +1,75 @@
+2. Once installed, launch VS Code.
 
-1. Once installed, launch VS Code.
+3. Click the extensions menu at the bottom on the left side. Then search for **C#**, install it then press reload.
 
-1. Click the extensions menu at the bottom on the left side. Then search for **IIS Express**, install it then press reload.
+    ![VS Code install extension](images/VsCode/VsCodeExtension.png)
 
-    ![VS Code install extension](images/VsCode/1.png)
+## Creating your Umbraco project
 
-1. Download a fresh Umbraco installation from [https://our.umbraco.com/download/](https://our.umbraco.com/download/) then unzip and drag it into VS Code. Or use our Umbraco .NET Core template which will download Umbraco automatically. For a guide on how to do that, check the [templates installation guide](install-umbraco-with-templates.md).
+Follow the [Templates Guide](install-umbraco-with-templates.md) to create your project folder.
 
-    ![Fresh Umbraco installation](images/VsCode/2.png)
+## Configure VS Code to run the Umbraco project
 
-1. To launch the site press **CTRL+F5**, this will open a local version of Umbraco in your standard browser.
+Open your project folder in VS Code, you project will look something like:
+
+![Fresh Umbraco installation](images/VsCode/netcoreStructure.png)
+
+Now we need to tell VS Code how to run your project.
+
+Open the command pallete, you can use the shortcut `Ctrl+Shift+P`, and type in `Tasks: Configure` and select the `Tasks: Configure Task` option:
+
+![Configure task option](images/VsCode/ConfigureTask.png)
+
+Select "Create task.json from template"
+
+![Create task from template](images/VsCode/TaskJsonFromTemplate.png)
+
+Now select ".NET Core" as your template. 
+
+![Create .NET Core Template](images/VsCode/NetcoreTemplate.png)
+
+After this VS Code will have created a folder called `.vscode` that contains a file called `tasks.json`, it's this file that tells VS Code how to build your project.
+
+Now that we've told VS Code how to build your project, we need to tell it how to actually launch it, luckily VS Code can also do this for you. First click the little play button in the left side menu, and then click the "create a launch.json file" link.
+
+![Creating launch.json file](images/VsCode/creatingLaunchFile.png)
+
+This will prompt a menu to appear, select the 
+
+![Nectore launch task](images/VSCode/NetcoreTask.png)
+
+Now you'll see a green play button appear with a dropdown where ".NET Core Launch" is selected. If you navigate to the files section again, you'll see that another file has been created in the `.vscode` folder, this one called `launch.json`, it's this file that tells VS Code, to build your project, run it, and then open a browser when you press F5.
+
+With that you're ready to run! Press F5, or click the little green play button in the "Run and Debug" section to run your brand new Umbraco site locally.
```

Comment:
```suggestion
Open your project folder in VS Code, your project will look something like this:

![Fresh Umbraco installation](images/VsCode/netcoreStructure.png)

Now we need to tell VS Code how to run your project.

Open the command palette, you can use the shortcut `Ctrl+Shift+P`, and type in `Tasks: Configure` and select the `Tasks: Configure Task` option:

![Configure task option](images/VsCode/ConfigureTask.png)

Select "Create task.json from template"

![Create task from template](images/VsCode/TaskJsonFromTemplate.png)

Now select ".NET Core" as your template. 

![Create .NET Core Template](images/VsCode/NetcoreTemplate.png)

After this VS Code will have created a folder called `.vscode` that contains a file called `tasks.json`, it's this file that tells VS Code how to build your project.

Now that we've told VS Code how to build your project, we need to tell it how to actually launch it, luckily VS Code can also do this for you. First, click the little play button in the left side menu, and then click the "create a launch.json file" link.

![Creating launch.json file](images/VsCode/creatingLaunchFile.png)

This will prompt a menu to appear, select the 

![Nectore launch task](images/VSCode/NetcoreTask.png)

Now you'll see a green play button appear with a dropdown where ".NET Core Launch" is selected. If you navigate to the files section again, you'll see that another file has been created in the `.vscode` folder, this one is called `launch.json`, it's this file that tells VS Code, to build your project, run it, and then open a browser when you press F5.

With that, you're ready to run! Press F5, or click the little green play button in the "Run and Debug" section to run your brand new Umbraco site locally.
```